<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Music Band</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0">
          <item row="8" column="5">
           <widget class="QToolButton" name="tbDelSingl">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="QTableView" name="tvArtist"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Artists</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QComboBox" name="cbBandName">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QToolButton" name="tbAddBand">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QToolButton" name="tbAddArtist">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QToolButton" name="tbDelArtist">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="tbDelBand">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item row="8" column="4">
           <widget class="QToolButton" name="tbAddSingl">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3" colspan="3">
           <widget class="QTableView" name="tvSingles"/>
          </item>
          <item row="2" column="3" colspan="2">
           <widget class="QLineEdit" name="leFilter"/>
          </item>
          <item row="2" column="5">
           <widget class="QToolButton" name="tbClearFilterTrack">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbBandInfo">
            <property name="text">
             <string>Mysic band info</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Tracks</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab2">
       <attribute name="title">
        <string>Albums</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="opaqueResize">
              <bool>true</bool>
             </property>
             <property name="childrenCollapsible">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="gridLayoutWidget">
              <layout class="QGridLayout" name="gridLayoutLeft">
               <property name="rightMargin">
                <number>3</number>
               </property>
               <item row="7" column="5">
                <widget class="QToolButton" name="tbSaveAlbum">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>v</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0" colspan="3">
                <widget class="QTableView" name="tvDateRelease"/>
               </item>
               <item row="0" column="3">
                <widget class="QLabel" name="label_10">
                 <property name="text">
                  <string>Albums</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="lbCompany">
                 <property name="text">
                  <string>Company</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="3">
                <widget class="QLineEdit" name="leSeller">
                 <property name="dragEnabled">
                  <bool>false</bool>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="7" column="1">
                <widget class="QToolButton" name="tbAddRelease">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Seller:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="4">
                <widget class="QToolButton" name="tbBreakAlbum">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="3">
                <widget class="QComboBox" name="cBoxCompany"/>
               </item>
               <item row="2" column="3">
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>Album tracks</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Date release:</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="2">
                <widget class="QToolButton" name="tbRemRelease">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="3" colspan="3">
                <widget class="QComboBox" name="cBoxAlbum"/>
               </item>
               <item row="3" column="3" rowspan="3" colspan="3">
                <widget class="QTableWidget" name="tableWidgetTracks"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="gridLayoutWidget_2">
              <layout class="QGridLayout" name="gridLayoutRight" rowstretch="0,0,0,0,0,0,0,0,1">
               <property name="leftMargin">
                <number>3</number>
               </property>
               <item row="4" column="0">
                <widget class="QToolButton" name="tbAddTrack">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&lt;</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1" rowspan="5" colspan="2">
                <widget class="QTableView" name="tvAllTracks"/>
               </item>
               <item row="2" column="2">
                <widget class="QToolButton" name="tbClearFilter2">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QLineEdit" name="leFilterTab2"/>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>All tracks</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QToolButton" name="tbRemTrack">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&gt;</string>
                 </property>
                </widget>
               </item>
               <item row="7" column="0">
                <widget class="QToolButton" name="tbRemCurrentTrack">
                 <property name="minimumSize">
                  <size>
                   <width>30</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>x</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab3">
       <attribute name="title">
        <string>Shop</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Склад</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="2">
            <widget class="QToolButton" name="tbDel2store">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QToolButton" name="tbAdd2store">
             <property name="minimumSize">
              <size>
               <width>25</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QTableView" name="tvStore"/>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Отчет</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,1,4">
           <item row="2" column="0">
            <widget class="QPushButton" name="pbReport2">
             <property name="text">
              <string>Лидеры продаж ($)</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pbReport1">
             <property name="text">
              <string>Лидеры продаж (шт)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="leYear"/>
           </item>
           <item row="0" column="1" rowspan="3" colspan="2">
            <widget class="QPlainTextEdit" name="plainTeReport"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Продажа</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QPushButton" name="pbApply">
             <property name="text">
              <string>Оформить</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QTableView" name="tvSell"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>788</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
